### PR TITLE
Add visualization

### PR DESCRIPTION
**Pull request by guywilsonjr**
Before doing this review please:

1. Take 15 mintues to Follow the 5 part [Dash Tutorial](https://dash.plotly.com/introduction) and run it yourself. If you want to use
      - Use `pip install jupyter-dash` (If you have trouble ask me)

Please tell if this is good. Please ask questions about anything unclear.

### CODE REVIEW

File: visual/main.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import sys
+from typing import Dict, List
+
+import pandas
+from icecream import ic
+import pandas as pd
+import plotly as plotly
+from dash import dcc, Dash, html, Input, Output
+import plotly.express as px
+import plotly.graph_objects as go
```
Comment (RACHT16) [inline visual/main.py:12]: until here, all the necessary files have been imported, like pandas, plotly etc and even some which I have seen for the first time like, functools, icecream, logging, sys, typing

File: visual/main.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+def setup_logging():
+    logging.basicConfig()
+    logger = logging.getLogger(__name__)
+    ic.configureOutput(includeContext=True, outputFunction=logger.info)
+    logger.addHandler(logging.StreamHandler(sys.stdout))
+    return logger
+    # logging.basicConfig(filename='example.log', encoding='utf-8', level=logging.DEBUG)
+
+logger = setup_logging()
```
Comment (RACHT16) [inline visual/main.py:23]: This is part is for setting up logs and warning, while writing the code.

File: visual/main.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+df: pd.DataFrame = pd.read_csv('playerInfo2018.csv')
+
+
+dropdown_label_key: str = 'label'
+dropdown_value_key: str = 'value'
+default_feature: str = 'points'
+
```
Comment (RACHT16) [inline visual/main.py:30]: this imports the datafile and create dropdown list

File: visual/main.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        first_half = df[df[col] < half_quantile][col]
+        second_half = df[df[col] >= half_quantile][col]
+
+        fig = go.Figure()
+        fig.add_trace(go.Histogram(x=first_half))
+        fig.add_trace(go.Histogram(x=second_half))
+
+        fig.update_layout(barmode='stack')
+        return fig
+
```
Comment (RACHT16) [inline visual/main.py:112]: Main Application with functions,   the first function initializes and changes the histogram on the dropdown value, the second one is for creating the dropdown menu and the third creates the histogram itself and below is the constructor to initialize the appl.

#have added comments for the remaining part of the code.

**Comment by guywilsonjr** [comment]
Ok. I made more updates to the code review.

**Review by PKrupa94 — APPROVED** [review]

**Pull request merged**